UI/Pages/WhichKeyStages.cshtml

```diff
@@ -8,37 +8,42 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <form method="get">
-            <h1 class="govuk-heading-l">Which key stages do you need tutoring support for?</h1>
-            <p class="govuk-body">Select all of the key stages that apply</p>
-
             <govuk-checkboxes asp-for="Data.KeyStages">
-                @foreach (var item in Model.Data.AllKeyStages)
-                {
-                    <govuk-checkboxes-item id="@item.Name.ToString().ToSeoUrl()" value="@item.Name" checked="@item.Selected" data-testid="key-stage-name">@item.Name.DisplayName()</govuk-checkboxes-item>
-                }
+                <govuk-checkboxes-fieldset>
+                    <govuk-checkboxes-fieldset-legend is-page-heading="true" class="govuk-fieldset__legend--l">
+                        Which key stages do you need tutoring support for?
```

Comment:
Think that it should remain within the h1?

<h1 class="govuk-fieldset__heading">

https://design-system.service.gov.uk/components/fieldset/